example/src/main/kotlin/com/example/afterpay/checkout/CheckoutFragment.kt

```diff
@@ -38,6 +49,21 @@ class CheckoutFragment : Fragment() {
         )
     }
 
+    private lateinit var cashButton: CashPayKitButton
+
+    private val payKitInstance: CashAppPayKit?
```

Comment:
We should avoid fragments knowing about the content of parent activity. A better way is to have a ViewModel hold the paykit instance, and have the ViewModel exist on both the activity and fragment. The Dev app we have to test the SDK showcases this well: https://github.com/squareup/cash-paykit-dev-app-android/blob/main/app/src/main/java/app/cash/paykit/devapp/MainActivityViewModel.kt

Reply:
Adressed in [0f7aa50](https://github.com/afterpay/sdk-android/pull/184/commits/0f7aa5028efd3e53f90101b55809ce0cb44cd9db)